internal/analytics/logger_adapter.go

```diff
@@ -0,0 +1,19 @@
+package analytics
+
+import "github.com/sirupsen/logrus"
+
+type loggerAdapter struct {
+	log logrus.FieldLogger
+}
+
+func newLoggerAdapter(log logrus.FieldLogger) *loggerAdapter {
+	return &loggerAdapter{log: log}
+}
+
+func (l *loggerAdapter) Logf(format string, args ...interface{}) {
+	l.log.Infof(format, args...)
```

Comment:
I think it is applicable only for verbose logging of the Segment client, which is IMHO not really useful:

```
INFO[2022-07-11T16:49:09+02:00] buffer (0/250) {identify 3fb4a9d9-303b-461e-9107-cfd0676cd387 db3091c0-1962-4dd6-9818-03fa31dad0be  2022-07-11 16:49:09.461648 +0200 CEST m=+0.036727431 <nil> map[botkubeVersion:{dev none unknown}] map[]}  component="Analytics reporter"
INFO[2022-07-11T16:49:09+02:00] buffer (1/250) {group 4addec75-e021-44ea-ae14-ea6372c7fece db3091c0-1962-4dd6-9818-03fa31dad0be  9d5c1cdd-737a-47f8-82ca-09d1b64259d4 2022-07-11 16:49:09.461656 +0200 CEST m=+0.036735424 <nil> map[k8sVersion:v1.22.7+k3s1] map[]}  component="Analytics reporter"
```

Anyway, let me know if I should change `Info` to `Debug` 🙂

Reply:
I'm OK with both opts

Reply:
Alright, keeping it as it is then 👍